src/routes/(main)/tournaments/[id]/+page.svelte

```diff
@@ -5,21 +5,37 @@
 	import Matches from '$lib/components/tournament-page/TournamentMatches.svelte';
 	import Button from '$lib/components/common/LargeButton.svelte';
 	import RegistrationButton from '$lib/components/tournament-page/RegistrationButton.svelte';
+	import { scoreModeEnum, teamModeEnum } from '$lib/TournamentEnums';
 
 	export let data: {
 		tournament: db.FullyPopulatedTournament;
 		editPerms: boolean;
 		sessionUserTeam: db.TeamWithMembers | null;
 	};
 	let { tournament, editPerms } = data;
-	let { name, acronym, id, team_size } = tournament;
+	let { name, acronym, id, team_size, x_v_x_mode, team_mode, score_mode, banner_url, color } = tournament;
+
+	const tournamentSummary = `
+	${tournament.description}
+	Team Size ${tournament.team_size}, ${tournament.score_mode}
+	Team Size ${team_size}, ${x_v_x_mode}v${x_v_x_mode}
+	${teamModeEnum[team_mode]}
+	${scoreModeEnum[score_mode]}
+	`;
 
 	$: tournament = data.tournament;
 	$: sessionUserTeam = data.sessionUserTeam;
 </script>
 
 <svelte:head>
 	<title>{acronym}: {name}</title>
+
+	<meta name="twitter:card" content="summary" />
+	<meta property="og:title" content={name} />
+	<meta property="og:type" content="website" />
+	<meta property="og:description" content={tournamentSummary} />
+	<meta property="og:image" content={banner_url} />
```

Comment:
If you're using the banner for the tournament image, [change the `twitter:card` property to `summary_large_image`](https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started#:~:text=one%20of%20%E2%80%9Csummary%E2%80%9D%2C%20%E2%80%9C-,summary_large_image,-%E2%80%9D%2C%20%E2%80%9Capp%E2%80%9D%2C%20or%20%E2%80%9Cplayer)